afdko/Tools/SharedData/FDKScripts/buildCFF2VF.py

```diff
@@ -484,32 +481,53 @@ def buildMasterList(inputPaths):
     # value list for that key for all fonts, using the values from
     # the first font which has the key. When other fonts with the
     # key are encountered, the real values will overwrite the default values.
+    privateDictList = []
     numMasters = len(cff2FontList)
-    for i in range(numMasters):
-        curFont = cff2FontList[i]
-        pd = curFont.privateDict
-        for key, value in pd.rawDict.items():
-            try:
-                valList = bcDictList[key]
-                if isinstance(valList[0], list):
-                    lenValueList = len(value)
-                    for j in range(lenValueList):
-                        val = value[j]
-                        valList[j][i] = val
-                else:
-                    bcDictList[key][i] = value
-            except KeyError:
-                if isinstance(value, list):
-                    lenValueList = len(value)
-                    valList = [None] * lenValueList
-                    for j in range(lenValueList):
-                        vList = [value[j]] * numMasters
-                        valList[j] = vList
-                    bcDictList[key] = valList
-                else:
-                    bcDictList[key] = [value] * numMasters
+    curFont = cff2FontList[0]
+    isCID = curFont.isCID
+    if isCID:
+        numFDArray = len(cff2FontList[0].topDict.FDArray)
+    else:
+        numFDArray = 1
+    for fdIndex in range(numFDArray):
+        blendedPD = {}
+        privateDictList.append(blendedPD)
+        for i in range(numMasters):
+            curFont = cff2FontList[i]
+            if isCID:
+                pd = curFont.topDict.FDArray[fdIndex].Private
+            else:
+                pd = curFont.topDict.privateDict
+            for key, value in pd.rawDict.items():
+                try:
+                    valList = blendedPD[key]
+                    if isinstance(valList[0], list):
+                        lenValueList = len(value)
+                        if lenValueList != len(valList):
+                            print("Error: the value list for key %s in the\
```

Comment:
don't use backslashes; do it like in [line 310](https://github.com/adobe-type-tools/afdko/pull/279/files#diff-89aac374e111040bfeda036e0dbdf138R310)